client/src/editor/components/Draft/Draft.js

```diff
@@ -124,13 +136,49 @@ const Draft = ({
           })}
         </select>
       </div>
+      <div>
+        <form onSubmit={e => handleImageUrlSubmit(e)}>
+          <label>
+            {draft.coverImage
+              ? 'Add link to a cover image: '
+              : 'Change cover image: '}
+            <input
+              type="text"
+              onChange={e => handleImageUrlChange(e.target.value)}
+            />
+            <input type="submit" value="Submit" />
+          </label>
+        </form>
+      </div>
       <div>
         <button onClick={handleDeleteDraft}>DELETE DRAFT</button>
       </div>
       <div>
         <div>
           <button onClick={handlePublishClick}>PUBLISH ADVENTURE</button>
         </div>
+        <>
+          {draft.coverImage ? (
+            <div>
+              <div>{`Your current cover image: ${draft.coverImage}`}</div>
+              <div>
+                <img
+                  style={{
+                    position: 'relative',
```

Comment:
We should be consistent and use css modules for all styles

Reply:
fixed